src/main/resources/errors.yaml

```diff
@@ -186,6 +186,8 @@ request-errors:
       The table ${keyspace}.${table} defines the primaryKeys: ${primaryKeyColumns}.
       
       Resend the request with an updated filter.
+      
+      Note, DeleteMany works as truncate when filter is empty.
```

Comment:
tweaks : "Note: deleteMany without a filter truncates the table" - and change title to "...Update and Delete One" commands